frontend/src/components/features/settings/project-management/linear-install-button.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { useTranslation } from "react-i18next";
+import { I18nKey } from "#/i18n/declaration";
+
+interface LinearInstallButtonProps {
+  "data-testid"?: string;
+}
+
+export function LinearInstallButton({
```

Comment:
@openhands update this install button so that its using tanstack query for the post request instead of changing the window location